buddyboss/buddyboss-platform
# When a group is public and it has a forum, anyone can post to it, even though they haven't joined the group

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1302

### How to test the changes in this Pull Request:

1. Go to 'BuddyBOss >> Settings >> Forums' and change the forum slug.
http://prntscr.com/ttlkbq
2. Now open a Public group without joining the group and navigate to the Discussion tab.
3. See, you can add a reply to the discussion without joining the group.

### Proof Screenshots or Video
http://prntscr.com/ttr4op



### Other information:
